apps/web/src/app/(default)/admin/pizza/page.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+import { useState } from "react";
+import { Heading, Input, Label } from "react-aria-components";
```

Comment:
Om du vil ha stylet versjoner av disse så finner du de i `@/components/ui`.

```suggestion
import { Heading, Input, Label } from "react-aria-components";
```